.github/workflows/android.yml

```diff
@@ -151,11 +110,11 @@ jobs:
           keystore_alias: 'release-alias'
           keystore_alias_password: ${{secrets.RELEASE_ANDROID_PASSPHRASE}}
 
-      - name: Deploy to App Center
-        run: |
-          echo "deploying to ${{ env.APPCENTER_NAME }} ${{ env.INPUT_APK_PATH }}"
-          yarn add appcenter-cli@^2.*
-          npx appcenter distribute release --token "${{ env.APPCENTER_TOKEN }}" --app "${{ env.APPCENTER_NAME }}" --group "Collaborators" --file "${{ env.UNIVERSAL_APK_PATH }}" --release-notes "$(git log -1 --pretty=format:'%h %s')" --debug
+#      - name: Deploy to App Center
```

Comment:
restore this. we need to make release to the appcenter.
the latest release will be used as base for hot code push